### PR TITLE
add timeout for netd serial tests

### PR DESCRIPTION
**Pull request by cadmuxe**

### CODE REVIEW

**Comment by cadmuxe** [comment]
cc/ @MrHohn

File: config/jobs/kubernetes/sig-network/ci-e2e-gce-netd.yaml [hunk 1]
```diff
@@ -1096,7 +1096,7 @@ periodics:
       - --ginkgo-parallel=1
       - --provider=gce
       - --test_args=--ginkgo.focus=\[sig-network\].*\[Disruptive\]|\[sig-network\].*\[Serial\]|\[Feature:PerformanceDNS\] --minStartupPods=8
-      - --timeout=120m
+      - --timeout=300m
```
Comment (MrHohn) [inline config/jobs/kubernetes/sig-network/ci-e2e-gce-netd.yaml]: @krzyzacy I vaguely remembered kubetest timeout has to be 20 mins less than the bootstrap timeout, is that true anymore?

Comment (krzyzacy) [reply config/jobs/kubernetes/sig-network/ci-e2e-gce-netd.yaml]: (at least 20min less, so 340m vs 120m is actually valid, depend on what you want to do for cluster down)

Comment (cadmuxe) [reply config/jobs/kubernetes/sig-network/ci-e2e-gce-netd.yaml]: thanks, I have updated this config file so that all tests have 40 min buff.

Comment (MrHohn) [reply config/jobs/kubernetes/sig-network/ci-e2e-gce-netd.yaml]: Thanks, that makes sense.

**Pull request merged**